Segment_Tree_With_Lazi_Propagation/Segment_Tree_With_Lazy_Propagation.py

```diff
@@ -1,12 +1,16 @@
-'''Segment Tree with Lazy Propagation is used when you want to perform some operation within an array range with range updation. Let's us suppose you want to perform minimum operation in certain range[a:b+1] and after performing minimum operation you are updating array index in range [l:r+1]. If you apply brute force approach then you will end up with O(n^2) complexity. If you use segment tree then your algorithm complexity will be O(nlogn) i.e. more efficient than bruteforce approach.
+'''Segment Tree with Lazy Propagation is used when you want to perform some operation within an array range with 
+range updation. Let's us suppose you want to perform minimum operation in certain range[a:b+1] and after performing
+minimum operation you are updating array index in range [l:r+1]. If you apply brute force approach then you will end
+up with O(n^2) complexity. If you use segment tree then your algorithm complexity will be O(nlogn) i.e. more efficient
+than bruteforce approach.
 
 '''
 
 INF = 99999
 tree = [0]*1000
 lazy = [0]*1000
 
-#Build Segment Tree
+# Build Segment Tree
 def build_tree(arr, node, a, b):
```

Comment:
Give meaningful names to variables. Instead of using `a, b` use `start, end` or something similar

Reply:
Yeah! I know it is not meaningful but i have describe there what is `a` and `b` in every function.

Reply:
Done